simdat.py

```diff
@@ -107,29 +116,95 @@ def drawraster ():
       plt.plot([pair[0]],[pair[1]],'ko',markersize=10)
     plt.xlabel('Time (ms)',fontsize=dconf['fontsize']); plt.ylabel('ID',fontsize=dconf['fontsize'])
 
-def calcerr (ddat):
+def calcerr (ddat, tstop, tstart=0.0):
   # calculates RMSE error from ddat dictionary
-  try:
-    NSig = errtot = 0.0; lerr = []
-    ddat['errtot']=None; ddat['lerr']=None
-    for fn,dat in ddat['dextdata'].items():
-      shp = dat.shape
-      # first downsample simulation timeseries to 600 Hz (assumes same time length as data)
-      dpldown = signal.resample(ddat['dpl'][:,1], len(dat[:,1]))
```

Comment:
The old way does not account for when exp data (dat[:,1]) is longer than simulation data.